Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -131,18 +133,19 @@ Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to `
 
    rm -rf -- /tmp/cilium-olm.tgz "/tmp/cilium-olm-${cilium_olm_rev}"
 
-.. note::
+At this stage manifest directory contains all that is needed to install Cilium.
+To get a list of the Cilium manifests, run:
 
-   At this stage manifest directory contains all that is needed to install Cilium.
-   To get a list of the Cilium manifests, run ``ls ${CLUSTER_NAME}/manifests/cluster-network-*-cilium-*``.
+.. code-block:: shell-session
 
-.. note::
+   ls "${CLUSTER_NAME}/manifests/cluster-network-*-cilium-*"
```

Comment:
double quotes turn `*` into a literal character, so the glob won't expand...

Reply:
I don't think they do, single-quotes do that

Reply:
I think Ilya is right, try `ls "*"`.

Reply:
:exploding_head: I could've have sworn that bash would expand the glob...sorry folks :sweat_smile: